hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/replication/TestReplicationManager.java

```diff
@@ -1740,6 +1742,16 @@ public void testInflightReplicationLimit() throws IOException {
         (int) Math.ceil(healthyNodes
             * config.getDatanodeReplicationLimit() * 0.75),
         rm.getReplicationInFlightLimit());
+
+    String hour =LocalDateTime.now().format(DateTimeFormatter.ofPattern("HH"));
+    config.setInflightOffPeakHour(hour);
+    config.setInflightOffPeakRatio(2);
+    configuration.setFromObject(config);
+    rm = createReplicationManager();
+    assertEquals(
+            (int) Math.ceil(healthyNodes
+                    * config.getDatanodeReplicationLimit() * 0.75 * 2),
```

Comment:
The indent used in Ozone is 4 blank spaces. You can set it into Intellij configuration.